src/uu/mv/src/mv.rs

```diff
@@ -319,24 +354,29 @@ fn handle_multiple_paths(paths: &[PathBuf], b: &Behavior) -> UResult<()> {
     let target_dir = paths.last().unwrap();
     let sources = &paths[..paths.len() - 1];
 
-    move_files_into_dir(sources, target_dir, b)
+    move_files_into_dir(sources, target_dir, opts)
 }
 
-fn exec(files: &[OsString], b: &Behavior) -> UResult<()> {
-    let paths = parse_paths(files, b);
+/// Execute mv command, moving 'source' to 'target', where
+/// 'target' is a directory. If 'target' does not exist, and source is a single
+/// file or directory, then 'source' will be renamed to 'target'.
+///
+/// returns MvError | UError
+pub fn exec_mv(files: &[OsString], opts: &Options) -> UResult<()> {
```

Comment:
Let's rename this to `move` or `mv`. I think that makes more sense for nushell.

Reply:
Ok I think move is a reserved keyword it will have to be mv :+1:

Reply:
Perfect!